model/src/main/java/com/javahelp/model/util/json/ClientUserInfoConverter.java

```diff
@@ -0,0 +1,58 @@
+package com.javahelp.model.util.json;
+
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.UserType;
+
+import jakarta.json.Json;
+import jakarta.json.JsonObject;
+
+/**
+ * Converts {@link com.javahelp.model.user.ClientUserInfo} to {@link jakarta.json.JsonObject}
+ */
+public class ClientUserInfoConverter implements IJSONConverter<ClientUserInfo> {
+
+    private static ClientUserInfoConverter instance = new ClientUserInfoConverter();
+
+    /**
+     * Private constructor
+     */
+    private ClientUserInfoConverter() {
+
+
+    }
+
+    /**
+     * @return {@link ClientUserInfoConverter} instance
+     */
+    public static ClientUserInfoConverter getInstance() {
+        return instance;
+    }
+
+    @Override
```

Comment:
This is also good, but I will update `Gender` attribute when this PR is merged first before the `Gender` PR